### PR TITLE
User account

### PR DESCRIPTION
**Pull request by eddybrando**
related: https://github.com/Qiskit/platypus/issues/258

preview: https://platypus-pr-174.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/signin

### CODE REVIEW

**Comment by vabarbosa** [comment]
@JRussellHuffman here is the feature branch for the user acct updates.
the preview URL: https://platypus-pr-174.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/account

**Comment by JRussellHuffman** [comment]
Wonderful, thank you! Do these links expire?

**Comment by vabarbosa** [comment]
> Wonderful, thank you! Do these links expire?

not until the PR is closed

**Comment by JRussellHuffman** [comment]
Good to know. 
More specifically, I'm thinking about doing some user testing and showing the user account page to some users just to validate the work so far. Ideally we use live code (ie that URL) instead of a clickable prototype. Any reason I shouldn't use that preview URL?

**Comment by vabarbosa** [comment]
> Good to know. More specifically, I'm thinking about doing some user testing and showing the user account page to some users just to validate the work so far. Ideally we use live code (ie that URL) instead of a clickable prototype. Any reason I shouldn't use that preview URL?

no reason at all, unless it is missing something you need (or including something you dont want) in the user testing

**Comment by vabarbosa** [comment]
draft PR can now be previewed so removing the `[WIP]` and putting into `Draft` so not to merge before ready

File: frontend/vue/components/SignIn/SignIn.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
+<template>
+  <section class="sign-in">
+    <div class="sign-in__content">
+      <div class="sign-in__content-section sign-in__content-section__hero">
```
Comment (techtolentino) [inline frontend/vue/components/SignIn/SignIn.vue:4]: I think we could remove `-section` from this class...  and stick w/ `sign-in__content__hero`, and in the next line `sign-in__content__hero-title`

File: frontend/vue/components/UserAccount/CourseProgress.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="course-progress">
+    <h5 class="course-progress__title">
+      {{ course.title }}
+    </h5>
+    <ColumnFlowGrid
+      class="course-progress__section-list"
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/CourseProgress.vue:7]: I don't see this class being used, maybe we could remove it?

File: frontend/vue/components/UserAccount/CourseProgress.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      :elements="course.sections"
+    >
+      <template #default="slotProps">
+        <SectionProgress
+          class="course-progress__section"
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/CourseProgress.vue:12]: I also don't see this class being used, maybe we could remove it?

File: frontend/vue/components/UserAccount/PrivacySection.vue [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<template>
+  <section class="privacy-section">
+    <h1 class="privacy-section__title">
+      Privacy
+    </h1>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_general">
+      IBM Privacy Statement
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/PrivacySection.vue:7]: we should add translation here and define the string in `strings.yaml`

File: frontend/vue/components/UserAccount/PrivacySection.vue [hunk 5]
```diff
@@ -0,0 +1,103 @@
+<template>
+  <section class="privacy-section">
+    <h1 class="privacy-section__title">
+      Privacy
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/PrivacySection.vue:4]: this is defined in `strings.yaml`, we can use 

`{{ $translate('Privacy') }}`

File: frontend/vue/components/UserAccount/PrivacySection.vue [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    </h1>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_general">
+      IBM Privacy Statement
+    </h4>
+    <AppLink class="privacy-section__policy-link" url="https://www.ibm.com/privacy">
+      https://www.ibm.com/privacy
+    </AppLink>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_quantum">
+      IBM Quantum End User Agreement
```
Comment (techtolentino) [inline frontend/vue/components/UserAccount/PrivacySection.vue:13]: same comment, we should add translation here and define the string in `strings.yaml`

**Review by techtolentino — CHANGES_REQUESTED** [review]
Just a few simple comments from me... 👏🏽

**Review by techtolentino — APPROVED** [review]

**Review by Tansito — APPROVED** [review]

**Pull request merged**